ddtrace/ext/ci.py

```diff
@@ -294,6 +294,12 @@ def extract_github_actions(env):
 def extract_gitlab(env):
     # type: (MutableMapping[str, str]) -> Dict[str, Optional[str]]
     """Extract CI tags from Gitlab environ."""
+    author = env.get("CI_COMMIT_AUTHOR")
+    author_name = None  # type: Optional[str]
+    author_email = None  # type: Optional[str]
+    if author:
+        author_name, author_email = author.strip("> ").split(" <")
```

Comment:
an example in a comment might make this easier to read, since this looks correct, but is weird to look at first time